backend/typescript/services/implementations/__tests__/campService.test.ts

```diff
@@ -0,0 +1,147 @@
+import db from "../../../testUtils/testDb";
```

Comment:
nice test! a nice to have could be a test case where creation fails to test if rollbacks work properly (would have to mock the creation of one of the models to fail)

Reply:
true yeah, I'll put this on the backlog